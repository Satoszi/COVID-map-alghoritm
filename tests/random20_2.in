20 20 3 87
random20_2
++..+|||+|.+.-|..+..
..|+.++..++|-.+-|+++
...+.-.++.++++++..++
|.+.-.+++.|+.+-.++.+
++..-+++..-+++.-.+++
..+-+++|-.++|+....++
.+..+.-+.+.-+++.+...
-|+.++..+++-.+.+-.|.
..+..+-+++|+++-+..+-
+.++.++++|.+..++++++
|-+-+++..+.+..-.+..+
+-|..++++.+.+++||++|
+|+..+|++-|..++.++-.
...+.+-....++.|+.-.|
-++.++.++.+.+..-.+++
+++++.|.|+.+---.++++
+|.+-+..+-+.--.++.++
+-.++..+++++++.+..+.
++.|++.|+.|--.++.+.+
.+|.+|.++.++.+.|.++.
