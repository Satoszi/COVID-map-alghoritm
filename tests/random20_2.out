13 19
11 19
2 19
1 19
19 18
17 18
14 18
11 18
5 18
3 18
13 17
7 17
1 17
18 16
16 16
12 16
10 16
5 16
11 15
9 15
6 15
4 15
15 14
10 14
7 14
0 14
19 13
17 13
15 13
14 13
11 13
6 13
3 13
8 12
6 12
2 12
1 12
19 11
18 11
16 11
15 11
12 11
10 11
14 10
11 10
9 10
4 10
14 9
0 9
7 8
18 7
16 7
4 7
2 7
0 7
16 6
9 6
6 6
1 6
18 5
10 5
8 5
2 5
15 4
4 4
16 3
10 3
4 3
2 3
0 3
19 2
8 2
5 2
16 1
12 1
3 1
2 1
17 0
14 0
13 0
11 0
9 0
8 0
7 0
5 0
4 0
0 0
